hatchet_sdk/hatchet.py

```diff
@@ -49,26 +52,31 @@ def workflow(
     default_priority: int | None = None,
     concurrency: ConcurrencyExpression | None = None,
     input_validator: Type[T] | None = None,
-) -> Callable[[Type[WorkflowInterface]], WorkflowMeta]:
+) -> Callable[[Type[TWorkflow]], WorkflowMeta]:
```

Comment:
important: getting rid of `WorkflowInterface` so that users don't see awkward type errors that should be mostly internal. we should revisit this `setattr` stuff later though